dashboard/src/containers/InstanceListViewContainer.ts

```diff
@@ -16,14 +16,14 @@ function mapStateToProps(
   const plans = catalog.plans;
   const classes = catalog.classes;
   const instances = catalog.instances;
-  const isInstalled = catalog.isInstalled;
+  const isServiceCatalogInstalled = catalog.isServiceCatalogInstalled;
```

Comment:
what do you think about using destructuring here to get these, e.g. `{ catalog: { brokers, plans, classes, instances, isServiceCatalogInstalled }, namespace }: IStoreState...`?

Alternatively, I think we can avoid the constants and just use the dot notation in the object below